micrometer-core/src/main/java/io/micrometer/core/instrument/binder/httpcomponents/DefaultApacheHttpClientObservationConvention.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.httpcomponents;
+
+import io.micrometer.common.KeyValues;
+import org.apache.http.HttpRequest;
+
+import java.util.function.Function;
+
+public class DefaultApacheHttpClientObservationConvention implements ApacheHttpClientObservationConvention {
```

Comment:
the `getContextualName()` should return sth like `HTTP Get` right? So `HTTP` word + HTTP Method of the given request?

Reply:
Right. Added that now.